pinot-connectors/pinot-flink-connector/pom.xml

```diff
@@ -51,7 +51,15 @@
     </dependency>
     <dependency>
       <groupId>org.apache.flink</groupId>
-      <artifactId>flink-streaming-java_${scala.compat.version}</artifactId>
+      <artifactId>flink-streaming-java</artifactId>
+      <version>${flink.version}</version>
```

Comment:
Please modify the version within the root pom file, and only reference them here

Reply:
IMO if flink is only being used inside this plugin it is ok to keep the versioning here. only when it is being used by a plugin and a core module (jackson for example) should the version be defined in root pom

Reply:
@walterddr It is already defined in the root pom, and we are overriding it here

Reply:
oh. ok. didn't know we moved it out.